### PR TITLE
Add support for the google-cloud-ops agent

### PR DESCRIPTION
**Pull request by rmoriar1**
Add support for the google-cloud-ops agent.

### CODE REVIEW

File: tasks/ops-agent.yml
Comment (rmoriar1) [inline tasks/ops-agent.yml:46]: Does the ops agent support additional configs? If so, in which directory are they stored?

Comment (rmoriar1) [reply tasks/ops-agent.yml:51]: Done.

File: tasks/main.yml
Comment (qingling128) [inline tasks/main.yml:4]: What about `logging`?

**Review by qingling128 — APPROVED** [review]
LGTM in advance as the pending comment is straight forward.

File: tasks/main.yml
Comment (rmoriar1) [reply tasks/main.yml:4]: That will be added in https://github.com/GoogleCloudPlatform/stackdriver-ansible-role/pull/41, I'll just have to merge the two accordingly.

File: tasks/ops-agent.yml
Comment (quentinmit) [inline tasks/ops-agent.yml:1]: With three files like this, these tasks are getting mighty redundant. Is it possible to refactor this so that there aren't three separate scripts and yaml files? (If there's a good reason they're separate, feel free to explain.)

**Review by quentinmit — CHANGES_REQUESTED** [review]

File: tasks/ops-agent.yml
Comment (rmoriar1) [reply tasks/ops-agent.yml:1]: Thats definitely possible, but it might be slightly less readable. We will need conditionals/variable substitution in lines 7, 23, 24, 32, 38, 45, 48 and 50.
Perhaps I can try that approach in a follow up PR after this gets merged?

Comment (rmoriar1) [reply tasks/ops-agent.yml:1]: Ling and I sync'd offline that this would be best handled in a follow-up PR. So i'm going to merge this one for the time-being. Will send out the refactored PR by EOD.

**Comment by koi8** [comment]
Now it fails with error:
 FAILED! => {"reason": "We were unable to read either as JSON nor YAML, these are the errors we got from each:\nJSON: Expecting value: line 1 column 1 (char 0)\n\nSyntax Error while loading YAML.\n  did not find expected key\n\nThe error appears to be in '/ansible/role/stackdriver-ansible-role/tasks/ops-agent.yml': line 29, column 3, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n\n  - name: Add repo and install agent or remove repo and uninstall agent\n  ^ here\n"}

So the block at the line 29 requires right indentation.

**Comment by rmoriar1** [comment]
> Now it fails with error:
> FAILED! => {"reason": "We were unable to read either as JSON nor YAML, these are the errors we got from each:\nJSON: Expecting value: line 1 column 1 (char 0)\n\nSyntax Error while loading YAML.\n did not find expected key\n\nThe error appears to be in '/ansible/role/stackdriver-ansible-role/tasks/ops-agent.yml': line 29, column 3, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n\n - name: Add repo and install agent or remove repo and uninstall agent\n ^ here\n"}
> 
> So the block at the line 29 requires right indentation.

Thanks for pointing this out. I believe the problem is the task at line 29 should not be indented. This will be fixed in https://github.com/GoogleCloudPlatform/stackdriver-ansible-role/pull/48.

**Pull request merged**
